core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalBackendConf.scala

```diff
@@ -53,6 +53,12 @@ trait InternalBackendConf extends SharedH2OConf {
         |  h2oClientLog         : ${h2oClientLogLevel}
         |  nthreads             : ${nthreads}
         |  drddMulFactor        : $drddMulFactor""".stripMargin
+
+
+  def setH2OLogLevel(level: String): H2OConf = {
```

Comment:
`setH2ONodeLogLevel` would be better here probably, just to make it more clear and consistent with the other method